pkg/master/master.go

```diff
@@ -118,11 +118,15 @@ type Config struct {
 	PublicAddress net.IP
 
 	// Control the interval that pod, node IP, and node heath status caches
+
 	// expire.
 	CacheTimeout time.Duration
 
 	// The name of the cluster.
 	ClusterName string
+
+	// Boolean indicating if we wish to create the ro/rw master services
+	SkipServices bool
```

Comment:
I see, I guess leave it but please add to the comment something about this being for testing only-- I don't want someone actually adding a command line flag to expose this.